connectors/cli.py

```diff
@@ -36,8 +36,20 @@ def _parser():
     parser.add_argument(
         "--action",
         type=str,
-        default=["schedule", "execute", "cleanup"],
-        choices=["schedule", "execute", "list", "config", "cleanup"],
+        default=[
```

Comment:
cc @vidok - do we need to be careful along with the CLI change when adding options here?